tests/build-std/main.rs

```diff
@@ -107,9 +115,12 @@ fn basic() {
         )
         .build();
 
-    p.cargo("check").build_std().target_host().run();
+    // An isolated CARGO_HOME environment is used elesewhere in this test
+    // to ensure no extra index updates is performed.
+    // It is achieve by asserting the complete stderr without any wildcard.
```

Comment:
```suggestion
    // HACK: use an isolated the isolated CARGO_HOME environment (`build_std_isolated`) to avoid `[BLOCKING]` messages (from lock contention with other tests) from getting in this test's asserts
```